src/routes/learn/outliers.svx

```diff
@@ -0,0 +1,72 @@
+---
+title: Outliers
+blurb: |
+    Some thinking about outliers and how to manage them.
+tags: 
+    - mean
+    - median
+    - mode
+    - normal
+    - histogram
+flair: tutorial
+---
+
+<script>
+    import Image from '$lib/components/Image.svelte';
+</script>
+
+## What is an outlier?
```

Comment:
Header capitalisation inconsistency.

To reduce noise I'll just flag this once here.